Podcast/SearchITunesTableViewController.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  SearchITunesTableViewController.swift
+//  Podcast
+//
+//  Created by Mindy Lou on 11/3/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+import NVActivityIndicatorView
+
+class SearchITunesTableViewController: ViewController, UITableViewDelegate, UITableViewDataSource, UISearchControllerDelegate, SearchSeriesTableViewDelegate, UISearchResultsUpdating, UISearchBarDelegate {
+
+    var searchResults: [Series] = []
+    let seriesCellIdentifier = "SeriesCell"
+    let seriesCellHeight: CGFloat = 95
+    
+    var loadingIndicatorView: NVActivityIndicatorView?
+    var tableView: EmptyStateTableView = EmptyStateTableView(frame: .zero, type: .search)
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tableView.frame = view.frame
+        tableView.showsVerticalScrollIndicator = false
+        tableView.separatorStyle = .none
+        tableView.delegate = self
+        tableView.dataSource = self
+        view.addSubview(tableView)
+        mainScrollView = tableView
+        
+        loadingIndicatorView = createLoadingAnimationView()
+        view.addSubview(loadingIndicatorView!)
+        loadingIndicatorView?.snp.makeConstraints { make in
+            make.center.equalToSuperview()
+        }
+        
+    }
+    
+    func fetchData(query: String, completion: (() -> ())?)  {
+        System.endpointRequestQueue.cancelAllEndpointRequestsOfType(type: SearchEpisodesEndpointRequest.self)
+        System.endpointRequestQueue.cancelAllEndpointRequestsOfType(type: SearchUsersEndpointRequest.self)
+        System.endpointRequestQueue.cancelAllEndpointRequestsOfType(type: SearchSeriesEndpointRequest.self)
+        
+        tableView.backgroundView?.isHidden = true
+        loadingIndicatorView?.startAnimating()
+        
+        let request = SearchITunesEndpointRequest(query: query, offset: 0, max: 20)
```

Comment:
is it pagenated now? if so we can add a moving offset

Reply:
nope, according to joe there's no way to paginate itunes search results